packages/explat/src/assignment.ts

```diff
@@ -22,7 +22,7 @@ export const fetchExperimentAssignment = async ( {
 		experiment_name: experimentName,
 		anon_id: anonId ?? undefined,
 		woo_country_code:
-			window.wcSettings.preloadSettings.general
+			window.wcSettings.admin.preloadSettings.general
```

Comment:
I think this will need to check for both objects 😅  

I know WC Pay bundles the ExPlat package directly at the moment, so in the future I think it would need to access the value from `window.wcSettings.preloadSettings.general` in situations where newer WC Pay is installed against an older version of WooCommerce.

Reply:
That's a good suggestion. Thank you for catching it 👍

Reply:
- Added change log in https://github.com/woocommerce/woocommerce-admin/pull/7600/commits/6b33b0361a477f5afe7a34fbd243b1752cf3331b
- Attempt to get the value in `window.wcSettings.preloadSettings.general` first in https://github.com/woocommerce/woocommerce-admin/pull/7600/commits/95243b3d162e3a38286321c34f57344fe2e1205c

I've also updated testing instructions.